scripts/copyPartytownFiles.ts

```diff
@@ -0,0 +1,70 @@
+import { ensureDirSync } from "https://deno.land/std@0.156.0/fs/mod.ts";
+import { join } from "https://deno.land/std@0.156.0/path/mod.ts";
+
+const partytownUrlPrefix = "https://unpkg.com/@builder.io/partytown@0/lib/";
```

Comment:
Do you think it's worth accepting an optional `version` param here, or `partytown` doesn't really look like something that updates a lot and have mismatch features?

Reply:
Since our import map uses version 0.x, the script follows the same major: 0.x.

Reply:
Not a big problem, just wondering. Looks good like this!